common-test/src/main/java/io/strimzi/test/k8s/BaseKubeClient.java

```diff
@@ -297,6 +312,14 @@ public String toString() {
         return asList(Exec.exec(namespacedCommand("get", resourceType, "-o", "jsonpath={range .items[*]}{.metadata.name} ")).out().trim().split(" +"));
     }
 
+    public List<String> listByLabel(String resourceType, String labelName) {
+        return asList(Exec.exec(namespacedCommand("get", resourceType, "-l", "name=" + labelName, "-o", "jsonpath={range .items[*]}{.metadata.name} ")).out().trim().split(" +"));
+    }
+
+    public String getConfig(String resourceType, String resourceName) {
```

Comment:
Why `getConfig` ? It executes a "get" command on any resource type with any name getting back the json ... so you are getting the JSON representation of a K8S resource.

Reply:
@ppatierno should I change getConfig to getResourceAsJson?

Reply:
to me it sounds better than `getConfig` ...

Reply:
Ok, thanks. Changed it